tests/integration/test_manifest_generation.py

```diff
@@ -498,6 +498,670 @@ def test_single_manifest_generation_google_sheet(
+        assert sheet1.freeze_panes == "A2"
+
+        # AND each of these cells in the first row has a comment "TBD"
+        for col in [
+            "Filename",
+            "Sample ID",
+            "File Format",
+            "Component",
+            "Genome Build",
+            "Genome FASTA",
+        ]:
+            assert sheet1[f"{columns[col]}1"].comment.text == "TBD"
+
+        # AND each of these cells in the first row do not have a comment
+        assert sheet1[f"{columns['entityId']}1"].comment is None
+
+        # AND the dropdown lists exist and are as expected
+        data_validations = sheet1.data_validations.dataValidation
+        file_format_validation = None
+        genome_build_validation = None
+        for dv in data_validations:
+            if f"{columns['File Format']}2" in dv.sqref:
+                file_format_validation = dv
+                continue
+            elif f"{columns['Genome Build']}2" in dv.sqref:
+                genome_build_validation = dv
+                continue
+            # AND there are no other data validations
+            assert False, f"Unexpected data validation found: {dv}"
+
+        assert file_format_validation is not None
+        assert file_format_validation.type == "list"
+        assert (
+            file_format_validation.formula1
+            == f"Sheet2!${columns['File Format']}$2:${columns['File Format']}$5"
+        )
+
+        assert genome_build_validation is not None
+        assert genome_build_validation.type == "list"
+        assert (
+            genome_build_validation.formula1
+            == f"Sheet2!${columns['Genome Build']}$2:${columns['Genome Build']}$5"
+        )
+
+        # AND the fill colors are as expected
+        for col in ["Filename", "Sample ID", "File Format", "Component"]:
+            assert sheet1[f"{columns[col]}1"].fill.start_color.index == LIGHT_BLUE
+
+        for col in [
+            "Genome Build",
+            "Genome FASTA",
+            "entityId",
+        ]:
+            assert sheet1[f"{columns[col]}1"].fill.start_color.index == GRAY
+
+        # AND conditional formatting is functioning as expected (MANUAL VERIFICATION)
+        workbook["Sheet1"][f"{columns['File Format']}2"].value = "BAM"
+        workbook["Sheet1"][f"{columns['File Format']}3"].value = "CRAM"
+        workbook["Sheet1"][f"{columns['File Format']}4"].value = "FASTQ"
+
+        # AND the workbook contains two sheets: "Sheet1" and "Sheet2"
+        assert workbook.sheetnames == ["Sheet1", "Sheet2"]
+
+        # AND the second sheet is hidden
+        assert sheet2.sheet_state == "hidden"
+
+        # AND the values in "Sheet2" are as expected
+        assert sheet2["A1"].value == "Filename"
+        assert sheet2["B1"].value == "Sample ID"
+        assert sheet2["C1"].value == "File Format"
+        assert sheet2["D1"].value == "Component"
+        assert sheet2["E1"].value == "Genome Build"
+        assert sheet2["F1"].value == "Genome FASTA"
```

Comment:
Could this ordering change 🤔 I would need to update my code as well. I haven't gotten any failures for this second sheet changing, so we're probably ok to assume it won't change.